script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -856,101 +976,137 @@ There's no way that nature could possess some sort of will.
 [sha:64cff31a1a83ed83b4db23745d560595303ddcef]{
 -- Page 217, Offset 22678.
 -- しかるに自己防衛なぞ、ましてや自分の身体を汚す害虫への敵意なんぞを抱くハズがない。
-Furthermore, there's no way it could act in self-defense, much less harbor animosity for the pests that have contaminated it.
+Even if it did,
+ there's no way it could act in self-defense,
+ much less harbor animosity for the pests that have contaminated it.
 }
 [sha:713a5cda09103f5293a0fc2d9588895eb2e506d8]{
 -- Page 218, Offset 22679.
 -- 『―――否。
-'No.
+// Guessing this is ya boi speaking, so italicizing this interlude
+'
+%{i}
+No
+%{/i}.
 }
 [sha:32d78456457d151ce99b847ac89b76ccd6973978]{
 -- Page 218, Offset 22680.
 -- それはある。確かにある。だがあまりにも生命としての段階、存在としての大きさが違いすぎて視えないだけだ。
-It does have one. Without a doubt, it does have one. You simply can't %{i}see%{/i} it because the size of its existence and the stage of its being are too far detached.
+%{i}
+It has one.
+ That much is certain.
+ It is simply a life that utterly dwarfs your own,
+ an existence so unfathomably vast that you cannot even hope to conceive of it.
+%{/i}
 }
 [sha:a8218c5d29639b504eb086102dc347868811f018]{
 -- Page 218, Offset 22681.
 -- この現実を見ろ。
-Behold, this reality.
+%{i}
+Look upon this reality.
+%{/i}
 }
 [sha:96ff47d4221ef40a6a37ee1a70c14310c25f02bf]{
 -- Page 218, Offset 22682.
 -- 実在する自然の感情を見ろ。
 // What am I even writing :cry:
-Behold, the emotions of nature in all of their actuality.
+%{i}
+Look upon these living, breathing, emotions of nature.
+%{/i}
 }
 [sha:45d999c458fc9f67c9302a81d4b39bd6de39515e]{
 -- Page 218, Offset 22683.
 -- 矮小な知覚には耐えられない色彩を見ろ。
-Behold, these colors that your puny perception cannot endure.
+%{i}
+Look upon these colors that your stunted sense of perception can scarcely endure.
+%{/i}
 }
 [sha:2e5247fc074debd525bb9b3b755c4cefb350f193]{
 -- Page 218, Offset 22684.
 -- この流動する生命の猛りと痛みを、おまえたちはまだ知らないだけだ』
-You humans do not yet know the pain and fury of this flowing lifeforce.'
+%{i}
+You humans are simply still unaware of the pain and fury of this flowing lifeforce.
+%{/i}
+'
 }
 [sha:743892a865ac5596f38e511a1f9661377ea5d790]{
 -- Page 219, Offset 22685.
 -- “あ―――、あ”
-'Ah―――― Ah'
+// Voiced, back to Shiki
+"%{i}Ah――Ah%{/i}..."
 }
 [sha:7b12ade45dc9c1ddc13b1b87e664f7a1a27ec0f9]{
 -- Page 219, Offset 22686.
 -- 一瞬の恐怖から解放される。
-I'm released from the momentary dread.
+I'm released from the sudden fear that had gripped me.
```

Comment:
Doesn't really do it for me without the second bit

Reply:
Well, the way it reads makes it feel more like the situation is the 恐怖 rather than him being scared (though given his choice of words, that's obviously true too)
You know, kinda like we use horror or terror in English sometimes.
>The horrors that lurk there!
>The terror of the universe.

Reply:
Not sure I totally understand what you're trying to say in the first sentence - I had interpreted this as him being gripped by a sudden feeling of fear (at having seen the blood lake), but are you suggesting it should be rendered more as him being visited by a 'horrible apparition' (the vision of the blood lake itself)?

Reply:
More like the experience itself was horror... the feeling of dread/terror/etc.
"I'm finally free from the horror of X" You know, those kinda statements.